packages/main-ui/src/pages/components/header.rs

```diff
@@ -73,6 +93,146 @@ pub fn Header(lang: Language, selected: i32) -> Element {
     }
 }
 
+#[component]
+pub fn MobileHeader(
+    #[props(default ="".to_string())] class: String,
+    lang: Language,
+    selected: i32,
+) -> Element {
+    let tr: HeaderTranslate = translate(&lang);
+    let mut popup: PopupService = use_context();
+    let current_path: Route = use_route();
+    let mut expanded = use_signal(|| false);
+    let toggle_menu = move |_| {
+        expanded.set(!expanded());
+    };
+    let mut close_menu = move || {
+        expanded.set(false);
+    };
+    let selected_menu = use_memo(move || match current_path {
+        Route::PoliticiansPage { .. } => 2,
+        Route::PoliticiansByIdPage { .. } => 2,
+        _ => 0,
+    });
+
+    let toggle_icon = if expanded() {
+        rsx! {
+            div { class: "w-[32px] h-[32px] flex justify-center items-center rounded-[4px] p-[10px] border-[1px] !border-[#464646]",
```

Comment:
```suggestion
            div { class: "w-32 h-32 flex justify-center items-center rounded-[4px] p-10 border !border-[#464646]",
```
input.css 에 컬러 정의된거나 정의하고 사용해주세요.

Reply:
넵 확인했습니다!